src/main/java/com/gmapp/app/altacontratos/VistaAltaContratos.java

```diff
@@ -285,199 +290,184 @@ public void comboClienteCCCremoveAllItem(){
         this.comboClienteCCC.removeAllItems();
     }
    
-    public void setEtqNIF(String etqNIF) {
-        this.etqNIF.setText(etqNIF);
+    public void setEtqNIF(String nif) {
+        etqNIF.setText(nif);
```

Comment:
a mí me gustan los this, dejan claro que trabajas con atributos y no elementos parciales :)